openupgrade_scripts/scripts/sale_timesheet/17.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 Tecnativa - Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def _fill_project_project_billing_type(env):
+    """Field `billing_type` on project.project is a new stored field."""
+    openupgrade.add_fields(
+        env,
+        [
+            (
+                "billing_type",
+                "project.project",
+                "project_project",
+                "selection",
+                False,
+                "sale_timesheet",
+            )
+        ],
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE project_project
+        SET billing_type = 'not_billable';
+        """,
+    )
```

Comment:
Anyway, being a static value, I'm not sure the gain doing this over the ORM counterpart.

```suggestion
                "not_billable",
            )
        ],
    )
```